File: src/main/java/ExampleScript.java
import org.powbot.api.loadout.LoadoutBuilder;
import org.powbot.api.script.*;

import com.warren.bt.BehaviorScript;
import com.warren.bt.BehaviorTree;

@ScriptManifest(name = "Example Script", description = "Example Script", version = "1")
public class ExampleScript extends BehaviorScript {
  @Override
  public void onStart() {
    super.onStart(); // Required
  }

  @Override
  public BehaviorTree.Builder builder() {
    //@formatter:off
    return BehaviorTree.builder()
      .sequence()
      .end();
    //@formatter:on
  }

  public static void main(String[] args) {
    new ExampleScript().startScript();
  }
}

File: src/main/kotlin/com/warren/example/ExampleScript.kt
package com.warren.example

// import com.warren.bt.*
import com.warren.bt.BehaviorScript
import com.warren.bt.BehaviorTree
import com.warren.bt.PowBehaviorTree
import com.warren.bt.Status
import org.powbot.api.rt4.Npc
import org.powbot.api.rt4.Npcs
import org.powbot.api.script.AbstractScript
import org.powbot.api.script.ScriptManifest

@ScriptManifest(
    name = "ExampleScript", description = "An example behavior script", version = "1.0.0", author = "Warren"
)
class ExampleScript : BehaviorScript() {
    override fun behaviorTree(): BehaviorTree {
        return PowBehaviorTree.build {
            sequence {
                succeed { logger.info("Hello, world") }
            }
        }
    }
}

fun main() {
    ExampleScript().startScript()
}
